apps/music/js/endpoint.js

```diff
@@ -267,27 +267,16 @@ function setShuffleSetting(shuffle) {
   return loadQueueSettings.then(() => PlaybackQueue.shuffle = shuffle);
 }
 
-function getAlbums() {
-  return new Promise((resolve) => {
-    Database.enumerateAll('metadata.album', null, 'nextunique', (albums) => {
-      resolve(albums);
-    });
-  });
+function getPlaylists() {
+  return Promise.resolve(Database.playlists);
```

Comment:
nit: It would be nice to make `Database.playlists` return a resolved `Promise` for consistency with the other methods.